R/api-bump-version.R

```diff
@@ -24,6 +24,13 @@
 #'
 #' @example man/examples/bump-version.R
 bump_version <- function(which = c("dev", "patch", "pre-minor", "minor", "pre-major", "major"), no_change_behavior = c("bump", "noop", "fail")) {
+
+  is_preamble_absent <- is.null(read_news()[["preamble"]])
```

Comment:
I put this here rather than in `bump_version_impl()`, but maybe that's not the right choice.

Reply:
Should the fledgeling object contain the info if a preamble is defined, and perhaps a method to add/update it?

Reply:
@krlmlr do you mean I should undo the change in `read_fledgeling()` in favor of this?

Reply:
going to do that now